docs/2.6.0/docs/deploy-daml/infrastructure-architecture/disaster-recovery.rst

```diff
@@ -1,8 +1,8 @@
 .. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Disaster Recovery
-#################
+Disaster Recovery (DR)
+######################
```

Comment:
This is undoing @carrielaben-da's change in #171. I don't feel comfortable arbitrating here. Assuming this has been discussed off-band, I'd prefer if Carrie were the one approving.

Reply:
This matches a change she couldn't see in another branch and I have informed her of it.

Reply:
This is an improvement on readability for people skimming for HA/DR of which, I'm sure, there are plenty.